Makefile

```diff
@@ -19,6 +19,10 @@ buildir:
 clean:
 	make -C backend/$(BACKEND) clean
 
+.PHONY: k8s
+k8s:
```

Comment:
It is breaking because k8s should call `clean buildir` first (as the deprecated targets are doing)

Reply:
Wooops. That's embarrashing. Thanks!